.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+
+on:
+  schedule:
+    # every weekday at 9:00 AM
+    - cron: '0 9 * * 1-5'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Python
+        uses: actions/setup-python@v1
+      - name: Install Dependencies
+        run: pip install tox
+      - name: Create Release
```

Comment:
Does it make sense to have some test steps before release?

Reply:
might as well as a sanity check, changes failing tests shouldn't get into master in the first place tho

Reply:
can you do a release again from fork and paste the link? just want to see what it looks like.

Reply:
https://github.com/danabens/sagemaker-experiments/runs/418481803?check_suite_focus=true